hack/build-local-images.py

```diff
@@ -166,17 +176,18 @@ def add_to_context(context_dir, source, destination, container_destination):
     sytem at the correct destination.
     """
     debug("Adding file:\n\tfrom {}\n\tto {}\n\tincluding in container at {}".format(
-    	source,
```

Comment:
spurious diff

Reply:
It's actually removing tabs and replacing with spaces.

Reply:
Yes but please let's not do this sort of thing